File: generator/app/templates/app/src/main/java/com/example/base/baseapp/persistence/ExampleDataDatabase.java
package com.example.base.baseapp.persistence;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.example.base.baseapp.model.ExampleData;

@Database(entities = {ExampleData.class}, version = 1, exportSchema = false)
public abstract class ExampleDataDatabase extends RoomDatabase {
    public abstract ExampleDataDao exampleDataDao();
}

File: generators/app/templates/app/src/main/kotlin/com/example/base/baseapp/persistence/ExampleDataDatabase.kt
package com.example.base.baseapp.persistence;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.example.base.baseapp.model.ExampleData;

@Database(entities = [ExampleData::class], version = 1, exportSchema = false)
abstract class ExampleDataDatabase : RoomDatabase() {
    abstract fun exampleDataDao(): ExampleDataDao
}